### PR TITLE
Issue 3248: Make communityhome work again.

### PR DESCRIPTION
**Pull request by annando**
See issue https://github.com/friendica/friendica/issues/3248

### CODE REVIEW

File: mod/community.php [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
 function community_content(App $a, $update = 0) {
 
 	$o = '';
 
-	// Currently the community page isn't able to handle update requests
-	if ($update)
-		return;
-
-	if((get_config('system','block_public')) && (! local_user()) && (! remote_user())) {
+	if ((Config::get('system','block_public')) && (! local_user()) && (! remote_user())) {
```
Comment (Hypolite) [inline mod/community.php]: Standards: Can you please add a space after commas in the whole file?

File: mod/community.php [hunk 2]
```diff
@@ -92,9 +89,9 @@ function community_content(App $a, $update = 0) {
 }
 
 function community_getitems($start, $itemspage) {
-	if (get_config('system','community_page_style') == CP_GLOBAL_COMMUNITY)
+	if (Config::get('system','community_page_style') == CP_GLOBAL_COMMUNITY) {
 		return(community_getpublicitems($start, $itemspage));
```
Comment (Hypolite) [inline mod/community.php]: Standards: Can you please remove parentheses from system calls (return, echo, include)?

**Comment by annando** [comment]
@Hypolite The change is that at the beginning of the function the first "if" is removed.

**Comment by Hypolite** [comment]
Ah, gotcha, hopefully soon we won't have that kind of mix between formatting and actual changes anymore! 🤞

**Comment by annando** [comment]
@Hypolite You are the reason (for the more complicated pull requests) ;-)

**Comment by Hypolite** [comment]
I take full responsibility for trying to normalize Friendica's codebase 😎

**Pull request merged**